### PR TITLE
Close #2 - Feature/iteration 1 - Add Traveler, TravelerRepository, and DestinationRepository classes and basic styling

### LINKED ISSUE #2: Iteration 1 - Dashboard
I should see a dashboard page that shows me:
- [x] All of my trips (past, present, upcoming and pending)
- [x] Total amount I have spent on trips this year. This should be calculated from the trips data and include a travel agent’s 10% fee

### PR DESCRIPTION
**Pull request by Prkr93**
## Description
- Added 3 classes - Traveler, TravelerRepository, and DestinationRepository.
- Added basic HTML structure and basic styles to display data for trips.
- Added fetch request to pull data from local server.

## Motivation and Context
Completes Issues #2 - Iteration 1

## How has this been tested?
- Built out a testing suite for each class.
- Utilized terminal to run npm test for test data sets.
- Had an issue when converting from the test data to the fetched data, so tests are no longer passing unless you readd those test data to the class files.

## Screenshots (if appropriate):
Screen Shot 2022-03-08 at 8 51 36 AM

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -7,10 +7,30 @@
...
-  
-  <!-- Do not include the scripts.js or class files here - it is done by the webpack server -->
-  <script src="bundle.js"></script>
+  <!-- <img src="./images/turing-logo.png" alt="turing logo"> -->
+  <header>
+    <h1>Travel Tracker</h1>
+  </header>
+  <main>
+    <h2>Dashboard</h2>
+    <section id="myTrips">
```
Comment (BrendaSotel0) [inline dist/index.html:16]: Consider adding classes to your sections for CSS manipulation.

File: src/domUpdates.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  let today = new Date();
+  today = [today.getFullYear(), today.getMonth() + 1, today.getDate()].reduce((acc, val) => {
+    val += '';
+    if (val.length === 1) {
+      val = '0' + val;
+    }
+    return [...acc, val];
+  }, []).join('/');
+
+  let travelerPastTrips = user.trips.filter(trip => trip.date <= today);
```
Comment (BrendaSotel0) [inline src/domUpdates.js:27]: Very interesting on that reverse fat arrow, I would love to know the background on that.

Comment (Prkr93) [reply src/domUpdates.js:27]: Today is evaluated to the same format as the dates from the API, so we are able to compare them as less than/greater than.

**Review by BrendaSotel0 — COMMENTED** [review]
If you can get that testing bug fixed you're going to be in a really good spot.

**Pull request merged**